testing/test_main.py

```diff
@@ -0,0 +1,142 @@
+import os
+import sys
+from fastapi.testclient import TestClient
+import pytest
+import random
+import string
+
+
+from .main import app, frankenbert
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+client = TestClient(app)
+
+""" Testing all Gets"""
+
+def test_get_bot_scripts():
+    response = client.get("/select-all-from-bot-scripts")
+    assert response.status_code == 200
+
+def test_frankenbert():
+    test = 'A women walked her dog down the street and cop waved to her'
+    response = client.get(f'/frankenbert/{test}')
+    assert response.status_code == 200
+    response_body = response.json()
```

Comment:
I like the addition to the unit tests, there aren't many uses for unit tests for us.